src/script/user/UserRepository.ts

```diff
@@ -219,22 +221,13 @@ export class UserRepository {
 
   async loadUsers(): Promise<void> {
     if (this.isTeam()) {
-      if (this.isTeamTooLargeForBroadcast()) {
-        this.logger.warn(
-          `Availability not displayed since the team size is larger or equal to "${UserRepository.CONFIG.MAXIMUM_TEAM_SIZE_BROADCAST}".`,
-        );
-        return;
-      }
-
```

Comment:
This guard is not needed anymore, as we don't limit the status update in the same way as we used to.